src/Styleguide/Components/__tests__/SelectedExhibitions.test.tsx

```diff
@@ -1,63 +1,75 @@
-import { mount } from "enzyme"
-import React from "react"
-import { exhibitions } from "../../../Apps/__test__/Fixtures/SelectedExhibitions"
-import { Boot } from "../../../Router/Boot"
-import { SelectedExhibitions } from "../SelectedExhibitions"
-
-jest.mock("../../../Router/PreloadLink")
+/**
+ * FIXME:
+ * For some reason Jest is throwing `_dec is not a function` in this test
+ * and this test only, though Boot (and decorators) are being used in many places
+ */
```

Comment:
@alloy - how did you recently fix `_dec` is not a function? Wasn't it just a dependency update to Babel? It's odd that it's now back -- but only ephemerally and only right here 🤔